snippets/loobins-project-ingest/loobins-project-ingest.storm

```diff
@@ -0,0 +1,70 @@
+/*
+Name: loobins-project-ingest.storm
+Author: exc3l_one@protonmail.com
+Modified By: TODO
+Last Modified: 2024-06-28
+
+Description: Ingest the JSON representation of the LOOBins project to model the ou:techniques associated with built-in macOS binaries
+
+References: 
+    -- LOOBins Project Website: https://loobins.io/
+*/
+
+// Generate the LOOBins project ou:org node
+init {
+    {[ou:org=$lib.gen.orgByFqdn("loobins.io") 
+        :name="loobins"
+        :url="https://www.loobins.io/"
+        :desc="Living Off the Orchard: macOS Binaries (LOOBins) is designed to provide detailed information on various built-in macOS binaries and how they can be used by threat actors for malicious purposes."
+    ]}
```

Comment:
This subquery won't actually execute since there are no nodes in the pipeline here